Packs/CybleEvents/Integrations/CybleEvents/README.md

```diff
@@ -0,0 +1,42 @@
+Cyble Events is an integration which will help Existing Cyble Vision users. This integration would allow users to access
+the API available as part of Vision Licensing and integrate the data into XSOAR.
+
+## Configure Cyble Intel on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cyble Intel.
+3. Click **Add instance** to create and configure a new integration instance.
+
+# Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook. After you
+successfully execute a command, a DBot message appears in the War Room with the command details.
+
+This integration provides following command(s) which can be used to access the Threat Intelligence
+
+**!cyble-vision-fetch-iocs**
+| **Parameter** | **Description** | **Required** | 
+| --- | --- | --- |
+| from | Returns records started with given value | False | 
+| limit | Number of records to return (max 1000). Using a smaller limit will get faster responses. | False | 
+| start_date | Timeline start date in the format "YYYY-MM-DD". Need to used with end_date as timeline range. | False | 
+| end_date | Timeline end date in the format "YYYY-MM-DD". Need to used with start_date as timeline range. | False | 
+| type | Returns record by type like (CIDR, CVE, domain, email, FileHash-IMPHASH, FileHash-MD5, FileHash-PEHASH,
+FileHash-SHA1, FileHash-SHA256, FilePath, hostname, IPv4, IPv6, Mutex, NIDS, URI, URL, YARA, osquery, Ja3,
+Bitcoinaddress, Sslcertfingerprint). | False | 
+| keyword | Returns records for the specified keyword | False|
```

Comment:
The README structure of the commands should be as the following (this is just an example):
```
### pan-os-delete-address
***
Delete an address object


#### Base Command

`pan-os-delete-address`
#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| name | Name of the address to delete. | Required | 
| device-group | The device group for which to return addresses (Panorama instances). | Optional | 


#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| Panorama.Addresses.Name | string | Address name that was deleted. | 
| Panorama.Addresses.DeviceGroup | String | Device group for the address \(Panorama instances\). | 


#### Command Example
```!pan-os-delete-address name="address_test_pb"```

#### Context Example
```json
{
    "Panorama": {
        "Addresses": {
            "Name": "address_test_pb"
        }
    }
}
```

#### Human Readable Output

>Address was deleted successfully.

It would be nice if you can change to that format. To automatically generate that you can use the SDK generate-docs command, see how to use [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md)

Reply:
made the changes